### PR TITLE
Add Logout button

### LINKED ISSUE #3: Add "Logout" to the navbar
Our session system is pretty simple, we just add `email` and `name` session keys. A logout button should clear all sessions and send you to the root `/` path

### PR DESCRIPTION
**Pull request by theodowling**
Closes #3 

Please let me know if you want anything changed

### CODE REVIEW

File: lib/sorteios_web/controllers/session_controller.ex [hunk 1]
```diff
@@ -50,6 +50,12 @@ defmodule SorteiosWeb.SessionController do
     end
   end
 
+  def delete(conn, _) do
+    conn
+    |> configure_session(drop: true)
```
Comment (lubien) [inline lib/sorteios_web/controllers/session_controller.ex]: Hi, I'm curious why you used configure_session?

I was maybe thinking of [clear_session/2](https://hexdocs.pm/plug/Plug.Conn.html#clear_session/1) but I'm open to feedback

File: lib/sorteios_web/live/components.ex [hunk 2]
```diff
@@ -21,7 +21,9 @@ defmodule SorteiosWeb.Components do
               </a>
             </div>
             <!-- Right section on desktop -->
-            <div class="hidden lg:ml-4 lg:flex lg:items-center lg:pr-0.5"></div>
+            <div class="hidden lg:ml-4 lg:flex lg:items-center lg:pr-0.5">
+              <%= link "Logout", to: "/logout", method: :delete, class: "mt-3 inline-flex w-full items-center justify-center rounded-md border border-white bg-indigo-600 px-4 py-2 font-medium text-white shadow-sm hover:bg-indigo-700 hover:border-indigo-300 focus:outline-none focus:ring-2 focus:ring-indigo-500 focus:ring-offset-2 sm:mt-0 sm:ml-3 sm:w-auto sm:text-sm" %>
```
Comment (lubien) [inline lib/sorteios_web/live/components.ex]: Can we change `to: "/logout"` to use `Routes....` helper?

**Review by lubien — CHANGES_REQUESTED** [review]

File: lib/sorteios_web/controllers/session_controller.ex (hunk 1 above)
Comment (theodowling) [reply lib/sorteios_web/controllers/session_controller.ex]: I'll double check this again. I remember having some strange behaviour when trying this, but I might have just done something wrong. Will look into this a bit more, and provide feedback.

**Comment by lubien** [comment]
@theodowling let me know if you need help :)

File: lib/sorteios_web/controllers/session_controller.ex (hunk 1 above)
Comment (theodowling) [reply lib/sorteios_web/controllers/session_controller.ex]: I checked what was done for logout looking at the new phx.gen.auth additions. 
It looks like `configure_session(conn, renew: true)` and `clear_session/1` is used. See https://github.com/dashbitco/mix_phx_gen_auth_demo/pull/1/files#diff-28c37a2a5a2d64995f1482f63a688624695c4f12001389d11dbd03c1d86f5fbbR62

It works well, but let me know if you require any further changes to this.

**Comment by theodowling** [comment]
Apologies for the delays. Both these changes are now incorporated.

**Review by lubien — APPROVED** [review]

File: lib/sorteios_web/controllers/session_controller.ex (hunk 1 above)
Comment (lubien) [reply lib/sorteios_web/controllers/session_controller.ex]: Wow, nice find!

**Comment by lubien** [comment]
No worries on the delay, OSS is like this.

Thanks for contributing!

**Pull request merged**